raw-cms-app/src/modules/core/views/login-view/login-view.tpl.html

```diff
@@ -6,7 +6,7 @@
         <div class="flex-grow-1"></div>
       </v-toolbar>
       <v-card-text>
-        <v-form ref="loginForm">
+        <v-form ref="loginForm" @keyup.native.enter="login()">
```

Comment:
use the submit event. be sure that the button which starts the login is of type submit.

Reply:
Done